src/plugins/node-workspace.ts

```diff
@@ -14,33 +14,48 @@
 
 import * as semver from 'semver';
 import cu = require('@lerna/collect-updates');
-import {Package, PackageJson} from '@lerna/package';
+import {Package as LernaPackage, PackageJson} from '@lerna/package';
 import {PackageGraph, PackageGraphNode} from '@lerna/package-graph';
 import {runTopologically} from '@lerna/run-topologically';
 import {ManifestPlugin} from './plugin';
 import {ManifestPackageWithPRData, ManifestPackage} from '..';
 import {VersionsMap} from '../updaters/update';
-import {packageJsonStringify} from '../util/package-json-stringify';
+import {jsonStringify} from '../util/json-stringify';
 import {CheckpointType} from '../util/checkpoint';
 import {RELEASE_PLEASE} from '../constants';
 import {Changes} from 'code-suggester';
 import {ConventionalCommits} from '../conventional-commits';
 import {Changelog} from '../updaters/changelog';
 
-type PathPkgJson = Map<string, PackageJson>;
+class Package extends LernaPackage {
+  constructor(
+    public readonly rawContent: string,
+    location: string,
+    pkg?: PackageJson
+  ) {
+    super(pkg ?? JSON.parse(rawContent), location);
+  }
+
+  clone() {
+    return new Package(this.rawContent, this.location, this.toJSON());
+  }
+}
 
 export default class NodeWorkspaceDependencyUpdates extends ManifestPlugin {
   // package.json contents already updated by the node releasers.
   private filterPackages(
     pkgsWithPRData: ManifestPackageWithPRData[]
-  ): PathPkgJson {
-    const pathPkgs = new Map();
+  ): Map<string, string> {
+    const pathPkgs = new Map<string, string>();
     for (const pkg of pkgsWithPRData) {
       if (pkg.config.releaseType === 'node' && pkg.config.path !== '.') {
         for (const [path, fileData] of pkg.prData.changes) {
-          if (path === `${pkg.config.path}/package.json`) {
+          if (
+            fileData.content !== null &&
```

Comment:
I added this because `content` can be `null`, but is this really necessary?

Reply:
I'd expect it to always be non-null (which I was just cheating the typing with `fileData.content!`